src/llmcompressor/modifiers/smoothquant/base.py

```diff
@@ -339,3 +341,5 @@ def _calculate_smoothing_scales(
         scales = torch.where(weight_scales > 0.0, scales, activation_scales)
 
         return scales
+
+    model_config = ConfigDict(arbitrary_types_allowed=True)
```

Comment:
should this be done in the Modifier class we inherit from?

Reply:
Potentially, there are arguments for both